deps/verifier/src/sgx/claims.rs

```diff
@@ -89,14 +84,11 @@ pub fn generate_parsed_claims(quote: sgx_quote3_t) -> Result<TeeEvidenceParsedCl
         quote.report_body.attributes.xfrm
     );
     parse_claim!(quote_body, "mr_enclave", quote.report_body.mr_enclave);
-    parse_claim!(quote_body, "reserved2", quote.report_body.reserved2);
     parse_claim!(quote_body, "mr_signer", quote.report_body.mr_signer);
-    parse_claim!(quote_body, "reserved3", quote.report_body.reserved3);
     parse_claim!(quote_body, "config_id", quote.report_body.config_id);
     parse_claim!(quote_body, "isv_prod_id", quote.report_body.isv_prod_id);
     parse_claim!(quote_body, "isv_svn", quote.report_body.isv_svn);
     parse_claim!(quote_body, "config_svn", quote.report_body.config_svn);
-    parse_claim!(quote_body, "reserved4", quote.report_body.reserved4);
```

Comment:
Is the field `reserved4` deleted from the quote? The same question for other `reserved*`. I think if it is still there, we can keep it as-is.

Reply:
> I think if it is still there, we can keep it as-is.

the thinking was to drop claims from the token that have no use-case. can you think of any?

Reply:
@mythi I have no idea. But to sincerely try to given more detail about the evidence is our aim. It is there, so we parse it out.

Reply:
Yes. We can rename/split the field when it's defined in future. For now users might not use the claims in policy, but we can still leave them there